client/accept-invite/invite-header/index.jsx

```diff
@@ -16,15 +16,90 @@ import Gravatar from 'components/gravatar';
 export default React.createClass( {
 	displayName: 'InviteHeader',
 
-	getInvitationVerb() {
-		switch ( get( this.props, 'invite.meta' ) ) {
-			case 'viewer':
-			case 'follower':
-				return this.translate( 'view' );
+	getInviterName() {
+		return get(
+			this.props,
+			'inviter.name',
+			this.translate( 'User', { context: 'Placeholder text while loading an invitation.' } )
+		);
+	},
+
+	getInvitedYouText() {
+		let text = '';
+
+		const inviterName = (
+			<strong className="invite-header__inviter-name">
+				{ this.getInviterName() }
+			</strong>
+		);
+
+		switch ( get( this.props, 'invite.meta.role' ) ) {
+			case 'administrator':
+				text = this.translate(
+					'{{inviterName/}} invited you to administer:', {
```

Comment:
`User invited you to administer:` feels a bit strange. I'd prefer `User invited you to manage:`